src/book.test.js

```diff
@@ -2,22 +2,22 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/. 
 import { expect, describe, it } from 'vitest';
-import { Book } from './book';
+import { Book } from "./book";
+import { schema } from "./models/configuration";
 
-describe('Book model', () => {
+describe("Book model", () => {
     // TODO confirm that this is what a newly created book (that is, without any settings changed) should look like; I've copied the result from how it currently works assuming it's working as-intended
     const defaultBook = {
         inputpdf: null,
         password: null,
         duplex: false,
-        duplexrotate: true,
-        papersize: [595, 842],
+        duplexrotate: false,
+        papersize: [612, 792],
```

Comment:
The defaults within the class weren't really the defaults that we were using. This is because they are immediately overwritten on start by the form and book update logic. So the defaults you would see when just constructing the class were different than in real use. Since we aren't maintaining multiple sources of truth for settings and defaults anymore, they are true to life.

I went ahead and moved the defaults back to A4 and "duplex", since that seems to be what we were expecting?